operator/v1/types_kubeapiserver.go

```diff
@@ -35,6 +35,23 @@ type KubeAPIServer struct {
 
 type KubeAPIServerSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+
+	// eventTTL specifies the amount of time that the events are stored before being deleted.
+	//
+	// The value must be parseable as a time duration value;
+	// see <https://pkg.go.dev/time#ParseDuration>.
```

Comment:
We prefer not to use duration values anymore. Instead, create a int32 type, with units in the name

For example, this should be `eventTTLMinutes`.

We do this because not all clients are built in Go, and building a Go compatible duration parsing in other languages is not going to be fun

Reply:
thanks for reminding me. I've changed this to be configurable in minutes via int32.